plugins/woocommerce/includes/admin/helper/class-wc-helper.php

```diff
@@ -903,7 +903,12 @@ private static function _helper_subscription_activate() {
 			wp_die( 'Could not verify nonce' );
 		}
 
-		$activated = self::activate_helper_subscription( $product_key, $product_id );
+		$activated = false;
```

Comment:
Do we need this line, if the `try ... catch` sets the value of `$activated`? Same with `$deactivated`.

Reply:
[059d447](https://github.com/woocommerce/woocommerce/pull/40703/commits/059d44760a01980d244d1acb0eb679e64c863b42)